.github/workflows/_release.yml

```diff
@@ -181,6 +181,7 @@ jobs:
           NVIDIA_API_KEY: ${{ secrets.NVIDIA_API_KEY }}
           GOOGLE_SEARCH_API_KEY: ${{ secrets.GOOGLE_SEARCH_API_KEY }}
           GOOGLE_CSE_ID: ${{ secrets.GOOGLE_CSE_ID }}
+          GROQ_API_KEY: ${{ secrets.GROQ_API_KEY }}
```

Comment:
Is there a way we can get an API key with a higher rate limit window that we only use for running integration tests on releases?

Current release flow will fail because of rate limit errors in integration testing.

Reply:
I am looking into this.